cmd/changelog.go

```diff
@@ -0,0 +1,144 @@
+package cmd
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"math"
+	"os"
+	"strings"
+	"syscall"
+	"time"
+
+	"gopkg.in/src-d/go-git.v4/plumbing/storer"
+
+	"github.com/chanzuckerberg/bff/pkg/util"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	git "gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/plumbing/object"
+)
+
+func init() {
+	rootCmd.AddCommand(changelogCmd)
+
+	// TODO: changelogCmd.Flags().BoolP("breaking", "b", false, "Breaking release")
+}
+
+var changelogCmd = &cobra.Command{
+	Use:   "changelog next-version",
+	Short: "Generate changelog entries based on git history",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) != 1 {
```

Comment:
shouldn't we just calculate this, like we do for the other commands?

Reply:
I created an issue and Asana task for Nadia : https://github.com/chanzuckerberg/bff/issues/8